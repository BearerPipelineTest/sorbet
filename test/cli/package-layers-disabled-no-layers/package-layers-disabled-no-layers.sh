cd test/cli/package-layers-disabled-no-layers || exit 1

../../../main/sorbet --silence-dev-message --stripe-packages --max-threads=0 . 2>&1
