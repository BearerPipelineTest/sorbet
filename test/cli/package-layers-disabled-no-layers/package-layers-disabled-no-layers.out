specifies-layer/__package.rb:4: Packages may only specify layers when --stripe-package-layers is set https://srb.help/3717
     4 |  layer 'good_layer_name'
          ^^^^^^^^^^^^^^^^^^^^^^^
Errors: 1